### PR TITLE
Cherry pick 3308 onto 1.19 - Fix priority expander falling back to random although higher priority matches

### PR DESCRIPTION
**Pull request by nitrag**
Cherry pick #3308 onto 1.19

### CODE REVIEW

**Comment by nitrag** [comment]
@MaciekPytel you previously recommended I do this in #3308

File: cluster-autoscaler/version/version.go [hunk 1]
```diff
@@ -17,4 +17,4 @@ limitations under the License.
 package version
 
 // ClusterAutoscalerVersion contains version of CA.
-const ClusterAutoscalerVersion = "1.19.0-beta.1"
```
Comment (mwielgus) [inline cluster-autoscaler/version/version.go:20]: This should not be changed in a cherry-pick.

**Review by mwielgus — CHANGES_REQUESTED** [review]

File: cluster-autoscaler/version/version.go (hunk 1 above)
Comment (nitrag) [reply cluster-autoscaler/version/version.go:20]: Hmm, this wasn't me. Should I re-do this PR?

Comment (mwielgus) [reply cluster-autoscaler/version/version.go:20]: Just remove this part of the change.

**Comment by nitrag** [comment]
@mwielgus Fixed

**Review by mwielgus — APPROVED** [review]
/lgtm
/approve

**Comment by cablespaghetti** [comment]
@mwielgus @nitrag This seems to have been merged to the wrong branch. This is still broken on the 1.19 branch. :disappointed: 

https://github.com/kubernetes/autoscaler/blob/cluster-autoscaler-release-1.19/cluster-autoscaler/expander/priority/priority.go

**Pull request merged**